tests/testthat/test-utils.R

```diff
@@ -2,47 +2,52 @@ context("Utility functions work as expected")
 
 test_that("knitr_sidecar_prefix preconditions hold", {
 
-  # Ensure that rmarkdown assumptions still hold
-  rmd_path <- file.path(tempdir(), "test-utils.Rmd")
+  if (rmarkdown:::pandoc2.0()) {
```

Comment:
Is there a reason you’re using a non exported function instead of pandoc_available?

Reply:
What you're saying is more correct... It's not a question of a low Pandoc version available, but Pandoc being available at all. I'll make this change.